src/tests/components/FacebookLogin.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { FacebookTest } from '../../components/login/FacebookButton';
+
+describe('facebooklogin component', () => {
+  let wrapper;
+  const props = { isLoggedIn: true, facebookLoginUserAction: jest.fn() };
+  beforeEach(() => {
+    wrapper = shallow(<FacebookTest {...props} />);
+  });
+  it('should render correctly', () => {
+    expect(wrapper).toHaveLength(1);
+  });
+  it('should test set props', () => {
```

Comment:
@kakaemma awesome work! 
I was once advised by the TTL @dondrzzy that all tests should `expect` something. You should refactor the tests that do not have an `expect`